src/utils/date-time-formatter.js

```diff
@@ -699,17 +703,27 @@ DateTimeFormatter.prototype.extractTokenValue = function (dateTimeStamp) {
  * @return {Date} : Native JS Date
  */
 DateTimeFormatter.prototype.getNativeDate = function (dateTimeStamp) {
-    if (dateTimeStamp instanceof Date) {
-        return dateTimeStamp;
-    } else if (isFinite(dateTimeStamp) && !!this.format) {
-        return new Date(dateTimeStamp);
+    let date = null;
+    if (Number.isFinite(dateTimeStamp)) {
```

Comment:
What if there are date objects in data? How will that be handled?

Reply:
How can we have date object in data ?
Any example will be helpful.

Reply:
`const data = [{
        date: new Date(1991, 0, 0)
  }, { date: new Date(1992, 0, 0) }]`
If user passes data in this format